### PR TITLE
cas exits

### PR DESCRIPTION
**Pull request by johannbarbie**
resolves https://github.com/leapdao/leap-contracts/issues/208

### CODE REVIEW

File: contracts/ExitHandler.sol [hunk 1]
```diff
@@ -225,6 +234,19 @@ contract ExitHandler is IExitHandler, DepositHandler {
 
       currentExit = exits[utxoId];
 
+      // exits of deleted periods are not payed out but simply deleted
+      if (exitRoots[utxoId] > 0) {
+        uint32 timestamp;
+        (, timestamp,,) = bridge.periods(exitRoots[utxoId]);
+        if (timestamp == 0) {
+          // stake goes to governance contract
+          address(uint160(bridge.admin())).send(currentExit.stake);
```
Comment (troggy) [inline contracts/ExitHandler.sol]: why so? Wouldn't it be locked there forever?

Comment (johannbarbie) [reply contracts/ExitHandler.sol]: ah Yeah, needs an update to governance contract

File: test/exitHandler.js [hunk 2]
```diff
@@ -112,6 +112,8 @@ contract('ExitHandler', (accounts) => {
       bridge = await Bridge.at(proxy.address);
       data = await bridge.contract.methods.setOperator(bob).encodeABI();
       await proxy.applyProposal(data, {from: accounts[2]});
+      data = await bridge.contract.methods.setOperator(accounts[1]).encodeABI();
```
Comment (troggy) [inline test/exitHandler.js]: already set on lines 113-114

File: contracts/ExitHandler.sol (hunk 1 above)
Comment (johannbarbie) [reply contracts/ExitHandler.sol]: @troggy  done

**Pull request merged**